requirements.txt

```diff
@@ -1,5 +1,5 @@
-PyJWT==1.1.0
-slumber==0.7.1
+# Package requirements
+-e .
```

Comment:
This installs the package in 'editable' mode (`setup.py`'s "developer mode"). It installs requirements for the package specified in the `setup.py`'s `setup` function and (I believe) adds the package to `PYTHONPATH`.

I'm not sure how else one would test locally.